### PR TITLE
[20501] Rearrange use cases list

### PR DESCRIPTION
**Pull request by rsanchez15**
## Description

@Mergifyio backport 2.13.x 2.12.x 2.10.x 2.6.x 

## Contributor Checklist

- [ ] Commit messages follow the project guidelines. 
- [N/A] Code snippets related to the added documentation have been provided.
- [x] Documentation tests pass locally.
- [x] Applicable backports have been included in the description.

## Reviewer Checklist

- [x] The PR has a milestone assigned.
- [ ] The title and description correctly express the PR's purpose.
- [X] Check contributor checklist is correct.
- [x] CI passes without warnings or errors.

### CODE REVIEW

**Comment by rsanchez15** [comment]
@richiprosima please test docs

**Comment by cferreiragonz** [comment]
@richiprosima please test docs

**Review by cferreiragonz — DISMISSED** [review]

**Comment by JesusPoderoso** [comment]
@mergifyio backport 2.13.x

**Comment by JesusPoderoso** [comment]
@Mergifyio backport 2.12.x  2.10.x  2.6.x

**Comment by cferreiragonz** [comment]
@richiprosima please test docs

**Review by cferreiragonz — APPROVED** [review]

File: docs/fastdds/use_cases/tcp/tcp_use_case.rst [hunk 1]
```diff
@@ -25,6 +25,11 @@ networks. When configured, UDP transport will exclusively be used during the :re
 taking advantage of the more reliable TCP/SHM for the remainder of the communication process. Fast DDS offers
 an extremely straightforward implementation for this mode through an environment variable, XML profiles or via code.
 
+For a video demonstration showcasing a practical example of this configuration, please refer to:
+`Large Data communication with ROS 2
+<https://docs.vulcanexus.org/en/latest/rst/use_cases/large_data/large_data.html>`_.
+
```
Comment (JesusPoderoso) [inline docs/fastdds/use_cases/tcp/tcp_use_case.rst:31]:
```suggestion
```

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

**Comment by JesusPoderoso** [comment]
@richiprosima please test this

**Review by JesusPoderoso — APPROVED** [review]

**Pull request merged**